client/src/components/TaskReflectionModal/TaskReflectionModal.js

```diff
@@ -29,6 +29,10 @@ const TaskReflectionModal = ({ onSubmit, onClose, onDone }) => {
     return dots;
   };
 
+  const submitHandler = () => {
+    onDone(reflectionComments, chosenDot + 1);
```

Comment:
I would rename this to `onSubmit` instead of `onDone`

Reply:
Renamed.